components/server/src/init.ts

```diff
@@ -63,6 +63,24 @@ if (process.env.NODE_ENV === "development") {
 log.enableJSONLogging("server", process.env.VERSION, LogrusLogLevel.getFromEnv());
 
 export async function start(container: Container) {
+    const server = container.get(Server);
+    const port = 3000;
+    const app = express();
+
+    process.on("uncaughtException", function (err) {
+        // fix for https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/load-balancer-pick-first.ts#L309
+        if (err && err.message && err.message.includes("reading 'startConnecting'")) {
+            log.error("uncaughtException", err);
+        } else {
+            throw err;
```

Comment:
Should we log and do process.exit(1) here? Or Node.js will do it?

Reply:
node js does it, i.e. that is the same behavior as not having a handler